onboarding.md

```diff
@@ -0,0 +1,195 @@
+**3. Configure the default files and run the server**
+```bash
+# start the server
+npm run start
+```
+
+Upon running `npm run start`, your output should look similar to:
+
+```bash
+
+> email-notifier@6.2.0 start /<path_to>/email-notifier
+> WITH_SASL=0&&LD_LIBRARY_PATH=$PWD/node_modules/node-rdkafka/build/deps&& node app.js
+
+2019-06-03T07:23:27.850Z - info: Connection with database succeeded.
+2019-06-03T07:23:27.863Z - info: CreateHandle::connect - creating Consumer for topics: [topic-notification-event]
+2019-06-03T07:23:28.116Z - info: CreateHandle::connect - successful connected to topics: [topic-notification-event]
+healthcheck is listening on port 3080
+
+```
+
+##  4. <a name='RunningInsideDocker'></a>Running Inside Docker
+
+We use `docker-compose` to manage and run the `email-notifier` along with its dependencies with one command.
+
+>*Note:* Before starting all of the containers however, start the `mysql` container alone, to give it some more time to set up the necessary permissions (this only needs to be done once). This is a short-term workaround because the `central-ledger` (which is a dependency of `email-notifier`) doesn't retry it's connection to MySQL.
+
+
+**1. First run the mysql container, then run the test of the containers**
+```bash
+docker-compose up mysql #first time only - the initial mysql load takes a while, and if it's not up in time, the central-ledger will just crash
+
+npm run docker:up
+```
+
+This will do the following:
+* `docker pull` down any dependencies defined in the `docker-compose.yml` file
+* `docker build` the `email-notifier` image based on the `Dockerfile` defined in this repo
+* run all of the containers together (`central-ledger`, `ml-api-adapter`, `central-event-processor`, `email-notifier`)
+
+
+### 4.1 Handy Docker Compose Tips
+
+You can run `docker-compose` in 'detached' mode as follows:
+
+```bash
+npm run docker:up -- -d
+```
+
+And then attach to the logs with:
+```bash
+docker-compose logs -f
+```
+
+When you're done, don't forget to stop your containers however:
+```bash
+npm run docker:stop
+```
+
+##  5. <a name='Testing'></a>Testing
+
+We use `npm` scripts as a common entrypoint for running the tests.
+```bash
+# unit tests:
+npm run test:unit
+
+# integration tests
+npm run test:integration
+
+# check test coverage
+npm run test:coverage
+```
+
+### 5.1 Testing the `email-notifier` API with Postman
+
+>Note: Make sure you have installed Postman and cloned the `mojaloop/postman` repo, which contains all the required collections and environments. You can find detailed instructions for this in the [general onboarding guide](https://github.com/mojaloop/mojaloop/blob/master/onboarding.md#2-postman).
+
+
+#### Prerequisites:
+* `ml-api-adapter` and `central-ledger` services running (follow the [Running Inside Docker guide](#RunningInsideDocker) to get these services up and running)
+* _Optionally_, run `central-timeout` , `cental-settlement` as well.
+
+
+#### Running Example Requests
```

Comment:
This isn't all that relevant for the `email-notifier`. You can probably just remove this section.